packages/pyolite-kernel/src/worker.ts

```diff
@@ -326,16 +326,13 @@ export class PyoliteRemoteKernel {
    *
    * @param content The incoming message with the comm target name.
    */
-  async commInfo(content: any, parent: any) {
+  async commInfo(content: { target_name: string }, parent: any) {
```

Comment:
seems like we should hoist that `content` to a type.

Reply:
Might revert back to `any` for now and do a pass on the types separately on the other methods as well.